From 711fcedf6a93e21408609eec74729be5106c149e Mon Sep 17 00:00:00 2001
From: Adeel Arshad <adeel_arshad@mentor.com>
Date: Wed, 2 Nov 2016 15:43:57 +0500
Subject: [PATCH] add tinxml to list of LIBS for CapturePlayer

Frame analyzer capture player uses the tinyxml,
so the tinyxml library needs to be added to the
list of the libraries on which CapturePlayer
build is dependent.

Signed-off-by: Adeel Arshad <adeel_arshad@mentor.com>
---
 CodeXL/Components/Graphics/Server/CapturePlayer/SConscript | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CodeXL/Components/Graphics/Server/CapturePlayer/SConscript b/CodeXL/Components/Graphics/Server/CapturePlayer/SConscript
index f114c62..1c74794 100644
--- a/CodeXL/Components/Graphics/Server/CapturePlayer/SConscript
+++ b/CodeXL/Components/Graphics/Server/CapturePlayer/SConscript
@@ -67,7 +67,8 @@ env['LIBS'] = \
     "pthread",
     "dl",
     "X11",
-    "xcb"
+    "xcb",
+    "tinyxml"
 ]
 
 sources = [
-- 
1.9.1

